translator/translate/otel/pipeline/applicationsignals/translator.go

```diff
@@ -52,12 +52,24 @@ func (t *translator) Translate(conf *confmap.Conf) (*common.ComponentTranslators
 	}
 
 	translators := &common.ComponentTranslators{
-		Receivers:  common.NewTranslatorMap(otlp.NewTranslator(common.WithName(common.AppSignals), otlp.WithSignal(t.signal))),
+		Receivers:  common.NewTranslatorMap[component.Config, component.ID](),
 		Processors: common.NewTranslatorMap[component.Config, component.ID](),
 		Exporters:  common.NewTranslatorMap[component.Config, component.ID](),
 		Extensions: common.NewTranslatorMap[component.Config, component.ID](),
 	}
 
+	// Add OTLP receivers
+	otlps, err := otlp.ParseOtlpConfig(conf, common.AppSignals, "", t.signal, -1)
+	if err == nil {
+		for _, otlpConfig := range otlps {
+			translators.Receivers.Set(otlp.NewTranslator(
+				otlpConfig,
+				otlp.WithSignal(t.signal),
+				common.WithName(common.AppSignals)),
```

Comment:
nit: I don't think these are actually used, so it might be a bit misleading to keep them.